quality-metrics/qm-007-integration-tests.md

```diff
@@ -22,5 +22,5 @@ Integration testing is pinnacle to velocity and verifying community cookbooks do
 Pseudocode or actual code that can be used to automatically verify the rule and/or assign appropriate points.
 
 ```bash
-   [ $(find test/integration/default/ -name '*_spec.rb' | wc -l) -ge $(find recipes/ -name '*.rb' | wc -l ) ]
+   [ $(find test/integration/ -name '*_spec.rb' | wc -l) -ge $(find recipes/ -name '*.rb' | wc -l ) ]
```

Comment:
Maybe this is a sufficient test given the variations of the names of the subdirectories and the types of files that might be found?

`[ $(find test/ -type f | wc -l) -ge $(find recipes/ -name '*.rb' | wc -l ) ]`